deployments/container/nvidia-mig-manager-example.yaml

```diff
@@ -68,6 +70,10 @@ spec:
         hostPath:
           path: /sys
           type: Directory
+      - name: host-dev
+        hostPath:
+          path: /dev
+          type: Directory
```

Comment:
I think we want to allow people to run the MIG manager outside of GKE's native support for MIG. Otherwise, people can't migrate from e.g. EKS to GKE smoothly.